bricks.js

```diff
@@ -189,4 +189,23 @@ function drawpuzzle(nColumns, nRows) {
 	drawgaps(nColumns, nRows)
 }
 
-drawpuzzle(nColumns, nRows);
\ No newline at end of file
+drawpuzzle(nColumns, nRows);
+
+const puzzle1 = {'B1-1': 3, 'B4-1': 1, 'B2-2': 3, 'B1-3': 3, 'B4-3': 4};
+
+function drawpuzzletext(puzzle) {
+	for (let key in puzzle) {
+		const xcoord = parseInt(document.getElementById(key).getAttributeNS(null, 'x')) + brickwidth / 2;
+		const ycoord = parseInt(document.getElementById(key).getAttributeNS(null, 'y')) + .75 * brickheight;
```

Comment:
nice! My first instinct is to use a [group](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/g) and a corresponding [transform](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/transform) for both the brick and the text, but I'm having trouble thinking of a reason why that would be better, haha. Just something to keep in mind next time you have to colocate some items on the canvas...